WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/ProductPriceSettingsViewController.swift

```diff
@@ -333,17 +333,57 @@ private extension ProductPriceSettingsViewController {
     func configurePrice(cell: UnitInputTableViewCell) {
         let cellViewModel = Product.createRegularPriceViewModel(regularPrice: viewModel.regularPrice, using: CurrencySettings.shared) { [weak self] value in
             self?.viewModel.handleRegularPriceChange(value)
+            self?.setCellPriceAccessibility(price: cell.value, cell: cell)
         }
         cell.selectionStyle = .none
         cell.configure(viewModel: cellViewModel)
+        setCellPriceAccessibility(price: cell.value, cell: cell)
+    }
+
+    func setCellPriceAccessibility(price: String?, cell: UnitInputTableViewCell) {
+        let value = cell.value.isEmpty == false ? cell.value : NSLocalizedString("empty", comment: "Accessibility text for when a price is empty")
+        let currency = cell.value.isEmpty == false ? CurrencySettings.shared.currencyCode.rawValue : ""
+        let accessibilityText = String.localizedStringWithFormat(
+            NSLocalizedString(
+                "Price %@ %@",
```

Comment:
when there are more than 1 variable in a localizable string, we usually specify the order of the variables like `Price %1$@ %2$@` in this case, and it would be helpful to note what the variables are in the comment